### PR TITLE
Feedback for diagnostics tool topic

### PR DESCRIPTION
**Pull request by bfallonf**
From #8644 

cc @rlopez133 

Will verify over email.

### CODE REVIEW

**Comment by bfallonf** [comment]
Thumbs up given.

@openshift/team-documentation PTAL

File: admin_guide/diagnostics_tool.adoc [hunk 1]
```diff
@@ -218,14 +227,15 @@ using the provided *_health.yml_* playbook.
...
-Due to potential changes the health check playbooks could make to hosts, they
-should only be used on clusters that have been deployed using Ansible and using
-the same inventory file with which it was deployed. Changes mostly involve
-installing dependencies so that the checks can gather required information, but
-it is possible for certain system components (for example, `docker` or
-networking) to be altered if their current state differs from the configuration
-in the inventory file. Only run these health checks if you would not expect your
-inventory file to make any changes to your current cluster configuration.
+Due to potential changes the health check playbooks could make to the
+environment, the playbooks should only be run against clusters that have been
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/the playbooks should only be run against clusters that have been deployed using Ansible with the same inventory file used during deployment/you must run the playbooks against only clusters that you deployed using Ansible and use the same inventory file that you used during deployment.

File: admin_guide/diagnostics_tool.adoc [hunk 2]
```diff
@@ -218,14 +227,15 @@ using the provided *_health.yml_* playbook.
...
+deployed using Ansible with the same inventory file used during deployment. The
+changes consist of installing dependencies so that the checks can gather the
+required information. In some circumstances, additional system components (i.e.
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/components (i.e. `docker` or networking configurations)/components, such as `docker` or networking configurations,

File: admin_guide/diagnostics_tool.adoc [hunk 3]
```diff
@@ -218,14 +227,15 @@ using the provided *_health.yml_* playbook.
...
+`docker` or networking configurations) may be altered if their current state
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/may/can

File: admin_guide/diagnostics_tool.adoc [hunk 4]
```diff
@@ -218,14 +227,15 @@ using the provided *_health.yml_* playbook.
...
+differs from the configuration in the inventory file. You should only run these
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/You should only run these health checks if/Run these health checks only if

File: admin_guide/diagnostics_tool.adoc [hunk 5]
```diff
@@ -282,9 +292,10 @@ xref:../install_config/registry/deploy_registry_existing_clusters.adoc#storage-f
 |This set of checks verifies that Curator, Kibana, Elasticsearch, and Fluentd
 pods have been deployed and are in a `running` state, and that a connection can
 be established between the control host and the exposed Kibana URL. These checks
-will only run if the `openshift_logging_install_logging` inventory variable is set
-to `true`, to ensure that they are executed in a deployment where
-xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[cluster logging] has been enabled.
+only run if the `openshift_logging_install_logging` inventory variable is set to
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: /only run/run only

File: admin_guide/diagnostics_tool.adoc [hunk 6]
```diff
@@ -282,9 +292,10 @@ xref:../install_config/registry/deploy_registry_existing_clusters.adoc#storage-f
...
+`true`, to ensure that they are executed in a deployment where
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: I don't think you need the comma after true.

File: admin_guide/diagnostics_tool.adoc [hunk 7]
```diff
@@ -282,9 +292,10 @@ xref:../install_config/registry/deploy_registry_existing_clusters.adoc#storage-f
...
+xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[cluster
+logging] has been enabled.
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/has been/is

File: admin_guide/diagnostics_tool.adoc [hunk 8]
```diff
@@ -349,14 +360,14 @@ playbook. For example:
 openshift_disable_check=etcd_traffic,etcd_volume
 ----
 
-Alternatively, set any checks you want to disable as variables with
-`-e openshift_disable_check=<check1>,<check2>` when running the
-`ansible-playbook` command.
+Alternatively, set any checks to disable as variables with `-e
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: Are these variables or parameter values? Do you mean this?

Alternatively, specify the names of checks to disable as values of the `openshift_disable_check` parameter, as shown:

 ----
 # ansible-playbook -e openshift_disable_check=<check1>,<check2>
 ----

File: admin_guide/diagnostics_tool.adoc [hunk 9]
```diff
@@ -367,9 +378,9 @@ ifdef::openshift-origin[]
 endif::[]
 image via the Docker CLI.
 
-To do so, specify your cluster's inventory file and the *_health.yml_* playbook
-when running the following `docker run` command as a non-root user that has
-privileges to run containers:
+Specify your cluster's inventory file and the *_health.yml_* playbook when
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: Could we just say
As non-root user that has privileges to run containers, run the following command:

File: admin_guide/diagnostics_tool.adoc [hunk 10]
```diff
@@ -404,15 +415,18 @@ that your original SSH key file will be relabeled to something like
...
 see the `docker-run(1)` man page.
 
-Keep this in mind for these volume mount specifications ￼because it could have
-unexpected consequences. For example, if you mount (and ￼therefore relabel) your
-*_$HOME/.ssh_* directory, *sshd* will become unable to access your public keys
-to allow remote login. To avoid altering the original file labels, mounting a
-copy of the SSH key (or directory) is recommended.
+[IMPORTANT]
+====
+These volume mount specifications could have unexpected consequences. For
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/could/can

File: admin_guide/diagnostics_tool.adoc [hunk 11]
```diff
@@ -404,15 +415,18 @@ that your original SSH key file will be relabeled to something like
...
+example, if you mount (and ￼therefore relabel) the *_$HOME/.ssh_* directory,
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/mount (and ￼therefore relabel)/ mount, and therefore relabel,

File: admin_guide/diagnostics_tool.adoc [hunk 12]
```diff
@@ -404,15 +415,18 @@ that your original SSH key file will be relabeled to something like
...
+*sshd* becomes unable to access the public keys to allow remote login. To avoid
+altering the original file labels, mounting a copy of the SSH key (or directory)
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/mounting a copy of the SSH key (or directory) is recommended./ mount a copy of the SSH key or directory.

File: admin_guide/diagnostics_tool.adoc [hunk 13]
```diff
@@ -404,15 +415,18 @@ that your original SSH key file will be relabeled to something like
...
+is recommended.
+====
 
-You might want to mount an entire *_.ssh_* directory for various reasons. For
-example, this would allow you to use an SSH configuration to match keys with
-hosts or modify other connection parameters. It would also allow you to provide
-a *_known_hosts_* file and have SSH validate host keys, which is disabled by the
+You might mount an entire *_.ssh_* directory for various reasons. For example,
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: I think this is part of the previous note.

Despite the risks of mounting an entire *_.ssh_* directory, mounting the directory offers benefits:
* You can use an SSH configuration to match keys with hosts or modify other connection parameters.
* You can provide a *_known_hosts_* file and have SSH validate host keys. This behavior is disabled by the
 default configuration. To use it, add `-e ANSIBLE_HOST_KEY_CHECKING=True` to the `docker run` command.

**Review by kalexand-rh — COMMENTED** [review]
This is mostly picks and style issues. Take what's useful!

**Pull request merged**